<launch>

    <!-- PERCEPTION -->
    <node pkg="perception" type="perception.py" name="perception" output="screen">
        <!-- IP Address -->
        <param name="ip_address" value="192.168.43.84" />
        <param name="dt_perception" value="0.05" />
        <param name="prediction_activated" value="True" />
        <param name="mapping_activated" value="True" />
        <param name="visualization_activated" value="True" />
    </node>

    <!-- ESTIMATION -->
    <include file="$(find estimation)/launch/estimation.launch"></include>

    <!-- PREDICTION -->
    <include file="$(find prediction)/launch/prediction.launch"></include>

    <!-- PLANNING-->
    <include file="$(find path_planning)/launch/path_planning.launch"></include>

    <!-- CONTROL -->
    <include file="$(find control)/launch/control.launch"></include>

    <!-- VISUALIZATION -->
    <include file="$(find visualization)/launch/visualization.launch"></include>

</launch>
